apps/www/src/registry/default/plate-ui/comment-value.tsx

```diff
@@ -4,21 +4,23 @@ import React from 'react';
 import {
   CommentEditActions,
   CommentEditTextarea,
-  useCommentValue,
+  useCommentEditTextarea,
+  useCommentEditTextareaState,
 } from '@udecode/plate-comments';
 
 import { cn } from '@/lib/utils';
 import { buttonVariants } from '@/registry/default/plate-ui/button';
 import { inputVariants } from '@/registry/default/plate-ui/input';
 
 export function CommentValue() {
-  const { textareaRef } = useCommentValue();
+  const commentTextAreaState = useCommentEditTextareaState();
+  const { props: textareaProps } = useCommentEditTextarea(commentTextAreaState);
```

Comment:
Are these lines necessary? It looks to me like `CommentEditTextarea.tsx` is already adding those props through `createPrimitiveComponent`, so I wouldn't expect passing `textareaProps` to do anything here.

I might be misunderstanding what's going on here, though, so correct me if I'm wrong.

Reply:
>  It looks to me like CommentEditTextarea.tsx is already adding those props through createPrimitiveComponent, so I wouldn't expect passing textareaProps to do anything here.

For some reason I thought I tested it without passing `textareaProps` and it did not work, but I just retested and you are correct. Dropping.